ci-operator/config/jianzhangbjz/learn-operator/jianzhangbjz-learn-operator-master.yaml

```diff
@@ -0,0 +1,84 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+  operator-registry:
+    name: "4.7"
+    namespace: ocp
+    tag: operator-registry
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- dockerfile_path: build/Dockerfile.ci
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /tmp/learn-operator
+      - destination_dir: .
+        source_path: /tmp/bin
+  to: learn-operator
+- dockerfile_path: deploy/Dockerfile.src.ci
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: learn-oc-bin-image
+operator:
+  bundles:
+  - context_dir: manifests
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/olmqe/learn-operator@sha256:2ada1ed5ef172a404f33a831f4cdac1c97a3ab069ab64a3bc04f468b8545edea
+    with: learn-operator
+promotion:
+  excluded_images:
+  - operator-registry-base
```

Comment:
There is no image called `operator-registry-base`?

Reply:
I guess no need to public any images, remove it.